dist/ci/travis_script.sh

```diff
@@ -33,6 +33,7 @@ if test -z "$SUBTEST"; then
       bundle exec rails test:spider
       ;;
     linter)
+      bundle exec rake db:structure:verify
```

Comment:
>> Where would you put it?
> in a different test suite called something like database structure?

That would mean to have another travis test job... which means probably slows things down even more. For a 10 second task this does not sound reasonable.

>> And does it really matter?
> It would be run even if you put it in the wrong place, but ordering things is important ￼

Generally yes, but let's not overdo it.